docker-compose.postgres.yaml

```diff
@@ -0,0 +1,60 @@
+# PostgreSQL-enabled configuration override for the registry service
+services:
+  postgres-secrets:
+    image: debian:bullseye-slim
+    command:
+      - /bin/bash
+      - -c
+      - >- 
+        cd /var/secrets/warg &&
+        if [[ ! -f postgres-admin-password ]]; then
+          < /dev/urandom tr -dc '[:alnum:]' | head -c16 >postgres-admin-password
+        fi
+    volumes:
+      - warg-secrets:/var/secrets/warg:rw
+  db:
+    # NOTE: Note as of 2023-04-26 not all IaaS supports v15 for managed SQL
```

Comment:
(Non-blocking) Any reason in particular?

Reply:
The secrets volume, or the database version? I was just being pedantic about not having an easy to guess password for something that may be opening up a port on one's machine and showing how to not pass it as an environment variable, and also assumed that the project isn't concerned about running on a specific version range of PostgreSQL yet. Just wanted to make sure it would be compatible for anyone running on in the cloud, e.g., [DO](https://docs.digitalocean.com/products/databases/postgresql/details/limits/) (latest v14), [GCP](https://cloud.google.com/sql/docs/postgres/db-versions) (latest v14), [AWS](https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-versions.html) (latest v15), [Azure](https://learn.microsoft.com/en-us/azure/postgresql/single-server/concepts-version-policy#supported--postgresql-versions) (latest v14), etc.